src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -91,6 +91,7 @@ def load_arguments(self, _):
                    help='If true, assign endpoint URL for direct access.',
                    options_list=['--assign-endpoint', c.deprecate(target='--is-public', redirect='--assign-endpoint', hide=True)])
         c.argument('https_only', arg_type=get_three_state_flag(), help='If true, access app via https', default=False)
+        c.argument('enable_end_to_end_tls', arg_type=get_three_state_flag(), help='If true, enable end to end tls')
```

Comment:
Do you want to add preview tag for this property if swagger is preview?

Reply:
No, we need to GA this feature.